dm/dm/worker/subtask_test.go

```diff
@@ -526,7 +528,7 @@ func (t *testSubTask) TestSubtaskFastQuit(c *C) {
 	c.Assert(st.Stage(), Equals, pb.Stage_Stopped)
 }
 
-func (t *testSubTask) TestGetValidatorError(c *C) {
+func TestGetValidatorError(t *testing.T) {
```

Comment:
just like ut in source_work_test, only need to test error case, let i t do normal case

TestOperateValidatorError and TestValidatorStatus too

Reply:
address in https://github.com/pingcap/tiflow/commit/5726353f3b7d20b3d383a52d0fb91fc2313714dd